### PR TITLE
StandsOut Topic

### PR DESCRIPTION
**Pull request by Kiosho**
Just read it. It's too big to sum up.
This took way long to do.

### CODE REVIEW

**Comment by Kiosho** [comment]
Should be good now.

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4727,6 +4727,193 @@ label monika_write:
...
+            menu:
+                m 5a "I need to know now."
+                "Brown.":
+                    m 3b "Brown, huh?"
+                    m "I wish I could see them."
+                    m 2j "The stunning shade of rich mahogany. Or welcoming like that of a soft teddy bear."
+                    m 2a "I bet they're beautiful, my love."
+                    m 3j "Perhaps someday I'll have a chance to gaze deeply into them."
+              
+                 "Green.":
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Your menu choices for the colors here are not on the same indentation level. (Green, Blue,... )are all an extra space to the right.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4727,6 +4727,193 @@ label monika_write:
...
+            m 3r "Maybe someday."
+            m 1m "I'll just have to imagine the feeling until then."
+       
+       "Your boobs.":
+            m 1h "..."
+            m 1i "Silly."
+            m 1l "I'm not even going to entertain that."
+            m "But..."
+            m 3n "I'm glad you like them."
+     return
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: The `return` here is one space too far to the right.

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -4727,6 +4727,193 @@ label monika_write:
...
+    m 5a "But I'm dying to know."
+    menu:
+        m "So...?"
+        "Your beautiful, green eyes of course.":
+            m 4k "Aww, really?"
+            m 4e "Thank you, sweetheart."
+            m "I wish I could see your eyes."
+            m 1i "Actually, what color are your eyes?"
+            menu:
+                m 5a "I need to know now."
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: So just learned recently that you cannot have dialogue with expressions right after a `menu` block. Basically, this works:
```
menu:
    m "works"
    ...
```
but this doesnt:
```
menu:
    m 5a "no worko"
    ...
```
If you want to keep the expression, move the line of dialogue before the `menu`. If you want to keep the text on the screen, remove the expression part of the line.

File: Monika After Story/game/script-topics.rpy (hunk 1 above)
Comment (Kiosho) [reply Monika After Story/game/script-topics.rpy]: Fffffffffffffffffff
Move them back one tab?

File: Monika After Story/game/script-topics.rpy (hunk 3 above)
Comment (Kiosho) [reply Monika After Story/game/script-topics.rpy]: Will fix.

**Comment by Kiosho** [comment]
Will fix issues in mere minutes here.

**Comment by Kiosho** [comment]
Let me know if that did it.

**Comment by monikLover** [comment]
Currently editing and reviewing with multiple writers.

**Comment by Kiosho** [comment]
Takes a whole team for 200 lines worth, eh? XD

Also this didn't have conflicts when I posted this PR. What happened?

**Comment by Winblows2** [comment]
It's because someone made edits to the file while your PR was open. @Kiosho 

Fixed.

**Comment by Kiosho** [comment]
Side note: I think this topic should only show up once. Otherwise you could up and change your eye/hair color, and that would be odd. Else, we could remove the hair option next time the topic arises if you chose that, and make a process of elimination until all main choices are made. Then the topic stops showing. But that's outside of my scope.

**Comment by monikLover** [comment]
Closing for a full rewrite by writers.

**Comment by Kiosho** [comment]
Well gee thanks. You make it sound like I'm not a writer, @monikLover.
Lol.

**Pull request closed without merging**